### PR TITLE
Create brenda2of4war

### LINKED ISSUE #1555: learning git
## Adding Surat to the map

Hi everyone! This issue is to track the addition of Surat to the map, which can be found at:

- Longitude: 21.1702° N
- Latitude: 72.8311° E

### PR DESCRIPTION
**Pull request by brenda2of4**
2nd attempt
Follow up @githubteacher from Mozfest 2016
closes #1555

### CODE REVIEW

**Comment by brianamarie** [comment]
Hi @brenda2of4! Make sure there's a space after the colon before the longitude number. 👍

**Comment by lemonmuncher14** [comment]
IDKY.

File: _pins/brenda2of4war [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+github handle: brenda2of4
```
Comment (crichID) [inline _pins/brenda2of4war]: @brenda2of4 you will want to remove the space between `github` and `handle`!

File: _pins/brenda2of4war [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+latitude: 53.405082
+longitude:-2.592726
```
Comment (crichID) [inline _pins/brenda2of4war]: @brenda2of4 please add a space here after the `:`

**Pull request merged**